CHANGELOG.md

```diff
@@ -5,12 +5,58 @@ For every (necessary) backward-incompatible Garp update we create a new tag, wit
 
 (not entirely semver-compatible, we know, but historically more compatible with how we came to Garp version 3 in the first place)
 
+## Version 3.22.0
+
+Hostname validation has been enabled for `Garp_Form_Element_Email`. Only the local part of the email address was validated. Now domains (except localhost) are allowed, IP addresses are not allowed.
```

Comment:
Shouldn't localhost be allowed, though?

Reply:
When adding ALLOW_LOCAL (https://framework.zend.com/manual/1.12/en/zend.validate.set.html#zend.validate.set.email_address.hostnametype) `examplecom` is also valid. Those are typo's are the most common ones that we want the prevent.

I've added a unit test.

Reply:
Hmmm. 
But an email validator is not meant to protect the user from entering `examplecom`. It won't protect you from entering `gmial.com`, ideally that's a different component that can do smarter inspection.

Although I think it's a bit of a coupled responsibility it's probably fine to include this until we run into trouble.

Reply:
Something like this: https://github.com/maximeg/email_inquire